include/z64.h

```diff
@@ -286,12 +286,53 @@ typedef struct {
     /* 0x16 */ char     unk_16[0x02];
 } Input; // size = 0x18
 
-typedef struct {
-    /* 0x0000 */ char   unk_00[0x28];
+typedef struct
+{
+   /* 0x0000 */ s16 unk_0;
+   /* 0x0002 */ s16 unk_2;
+   /* 0x0004 */ u16 unk_4;
+   /* 0x0004 */ u16 unk_6;
+   /* 0x0008 */ s16 unk_8;
+   /* 0x000A */ s16 unk_A;
+   /* 0x000C */ u16 unk_C;
+   /* 0x000E */ u16 unk_E;
+} UNK_VIEW_STRUCT; // size = 0x10, probably a viewport in disguise
+
+typedef struct
+{
+   /* 0x0000 */ s32 topY;       //uly (upper left y)
```

Comment:
Comments should always have a space before their content, as it was decided not that long ago. This applies to other places in this file as well as in z_view.c.